language/libra-vm/src/libra_vm.rs

```diff
@@ -1047,6 +1048,12 @@ fn convert_txn_args(args: &[TransactionArgument]) -> Vec<Value> {
         .collect()
 }
 
+/// Get the AccessPath to a resource stored under `address` with type name `tag`
+fn create_access_path(address: AccountAddress, tag: StructTag) -> AccessPath {
+    let resource_tag = ResourceKey::new(address, tag);
+    AccessPath::resource_access_path(&resource_tag, &Accesses::empty())
+}
+
```

Comment:
This is identical to `make_access_path` above in e2e tests. I wonder if it makes sense to move this into `libra_types`?

Reply:
that's a very good point and I looked into it because I wanted to make the change, and now I am not sure anymore
I think it should go somewhere there but there is not a great location for it now. It could go with `ResourceKey` or `AccessPath` and I am not sure where to put it so we'll leave it here for now.
It seems pretty obvious that the dependency story on some of our types is not that well organized and maybe we should really think to rework that (for instance whether we need and AccountAddress in the VM as you mentioned in the past)